### ISSUE TITLE
Implement `secrets touch` command

### ISSUE THREAD

**Issue by tegefaulkes** [MatrixAI/Polykey-CLI#244]
### Specification

`touch` serves two purposes
1. Will create a new empty file if no file exists at that path.
2. Will update the timestamps of a file if it already exists.

For our `secrets touch` command we need to support these two actions while working with normal file paths and secret paths. It should also support multiple paths for touching multiple files. Wild cards are supported as well but will not create files.

So we have two levels of functionality.
1. Create files.
2. Update file access times without modifying contents.

I don't know if we want to include updating the files access times. Most of the commands complexity will come from that.

### Additional context

Related #32

### Tasks

1. Implement a `secrets touch` command.
4. Must create a new empty secret or file if no file exists at the provided path.
5. If secret or file exists then we update the updated and access time without modifying contents.
6. It takes a varadic argument of file paths, so we can touch multiple files at a time.
7. File paths and secret paths should be interchangeable. 
8. Touching to create a file can't operate recusively. So it won't create a directory for a touched file.
9. Wildcards can be used for updating access times of files, but not for creating files.

**Comment by CMCDragonkai** [comment]
Didn't this get implemented?

**Comment by aryanjassal** [comment]
> Didn't this get implemented?

No, not yet. I opened a PR, but I started working on the secrets env command. I will start work on this once the more pressing PRs have been closed.
